paasta_tools/cli/cmds/status.py

```diff
@@ -447,14 +609,16 @@ def apply_args_filters(args):
     return clusters_services_instances
 
 
-def paasta_status(args):
+def paasta_status(
+    args,
+) -> int:
     """Print the status of a Yelp service running on PaaSTA.
     :param args: argparse.Namespace obj created from sys.args by cli"""
     soa_dir = args.soa_dir
     system_paasta_config = load_system_paasta_config()
 
     if 'USE_API_ENDPOINT' in os.environ:
-        use_api_endpoint = strtobool(os.environ.get('USE_API_ENDPOINT'))
+        use_api_endpoint = strtobool(os.environ['USE_API_ENDPOINT'])
```

Comment:
I would be OK if you "forced" paasta status to use the API for k8s, no questions asked.

Reply:
+1, that was my thought too. It won't work any other way so no need to make the user specify it.

Reply:
Please note that this diff uses the single `paasta status` command for both Marathon/Mesos and k8s setups.  For Marathon/Mesos `paasta status` can use either ssh or REST API depending on `USE_API_ENDPOINT` environment variable value, for k8s setup `paasta status ` can use only REST API (when `USE_API_ENDPOINT` is true) and just doesn't work otherwise.
We can detect the current setup only after receiving a response from the REST API.
Said so, we cannot force `paasta status` to use REST API for k8s without forcing it for Marathon/Mesos as well.
Also note, that the REST API is inferior comparing to ssh, as it i.e. doesn't support `-v`.
So, either we use RESP API by default for both Marathon/Mesos and k8s setups (probably after extending REST API to support `-v` and other missing thing), or we introduce something like `paasta status_k8s` specifically for k8s.
What do you think?